packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -1075,3 +1080,63 @@ export function unmountFundamentalComponent(
 export function getInstanceFromNode(node: HTMLElement): null | Object {
   return getClosestInstanceFromNode(node) || null;
 }
+
+export function mountEventListener(listener: ReactDOMListener): void {
+  if (enableUseEventAPI) {
+    const {target} = listener;
+    if (target === window) {
+      // TODO
```

Comment:
nit IMO it's_ nice to tag TODOs with a little more info to make them project grepable, e.g. `TODO (useEvent)`